DiffieHellman.ab

```diff
@@ -19,7 +19,14 @@ Bob {
 Bob -> Alice: b = M2<gy>
 Alice {
     let M2<gy> = b;
-    let gxy = exp(gy, x);
+    let gyx = exp(gy, x);
+}
+Alice -> Bob: c = M3<gyx>
+Bob {
+    let M3<gyx> = c;
+    new t: key;
+    new f: key;
+    let result = if(gxy = gyx, t, f);
```

Comment:
Are you sure we want this in our Alice&Bob file for DiffieHellman?

Reply:
Any suggestions are welcome, I think I went with this because that was what we discusses 🤔

Reply:
Ahh, I misunderstood. No, this was just a test case, will remove